### ISSUE TITLE
Unable to bind PrintScreen and ISO Extra Key ( / | ) for UK Layout

### ISSUE THREAD

**Issue by Jacksaur** [rvaiya/keyd#65]
Having some trouble binding these two keys. My config is

pageup = home
pagedown = prtsc
rightshift = \

The Pageup key is changed correctly, but Pagedown has no effect. Curiously, it does type a Tilde if I press it while in a terminal. I couldn't see PrtSc mentioned anywhere in the keyd -l command, is it not usable?

And as for the Right Shift bind, my keyboard is set to UK Layout in my system settings, which seems to make the backslash key register as the Hash/Tilde key instead. I tried binding the key to Hash to see how that would affect things, and then it would only print a pound sign.
The binding works on Windows using SharpKeys, and it names the key I have it bound to as "ISO Extra Key 00_56". Any way I could bind that key properly as well?

**Comment by rvaiya** [comment]
> The Pageup key is changed correctly, but Pagedown has no effect. Curiously, it does type a Tilde if I press it while in a terminal. I couldn't see PrtSc mentioned anywhere in the keyd -l command, is it not usable?

I believe it should be named `sysrq` (you can use `keyd -m` to figure out key code named by pressing the key).

> And as for the Right Shift bind, my keyboard is set to UK Layout in my system settings, which seems to make the backslash key register as the Hash/Tilde key instead. I tried binding the key to Hash to see how that would affect things, and then it would only print a pound sign.

keyd operates at the level of the key code, it is completely ignorant of your display server remapping. The names used in the config file are based on a US/QWERTY labeled keyboard (which also the names used by the kernel). Thus the letter `q` on a french keyboard will actually generate what keyd considers an `a` since `q` on a french keyboard is in the same position as `a` on a US one.

You need to remap the key to whatever key code causes your display server to produce `\`. You have two options:

1) Use `keyd -m` to figure out what key code actually corresponds to the `\` on your keyboard and bind `rightshift` to that.

2) Switch to a US layout in your display server (this will make right hand values behave as expected, though left hand values still won't match the labels on your physical board).

TLDR: Someone forgot to inform the key codes that the sun never sets on the British Empire.

**Comment by Jacksaur** [comment]
Can't believe I missed keyd -m in the manpage, terribly sorry about that, it solved everything.
PrintScreen was sysreq as you suggested, and my backslash key was interestingly referred to as "102nd"

With those two codes, the bindings now work perfectly. Thanks!
